src/main/java/build/buildfarm/instance/server/NodeInstance.java

```diff
@@ -154,6 +154,7 @@ public abstract class NodeInstance extends InstanceBase {
   protected final DigestUtil digestUtil;
   protected final boolean ensureOutputsPresent;
 
+  private static final Netrc NETRC_INSTANCE = Netrc.getInstance();
```

Comment:
Rather negates the singleton principle if we have to keep a reference around.

Reply:
Yoops you're right. I was thinking of avoiding all the (re-)parsing of the default `$HOME/.netrc` by holding an instance var. 🤦